### ISSUE TITLE
Instagram Follows App API Error

### ISSUE THREAD

**Issue by IIAIronWolf** [tidbyt/community#2707]
The Instagram Follows app by @danielsitnik API Error code 429 and does not currently work.

**Comment by danielsitnik** [comment]
Yes. Unfortunately the Tidbyt server IP address is being blocked (rate limited) by Instagram.
I will have to ask for this app to be removed. 😢

**Comment by IIAIronWolf** [comment]
Oh nooo. I suppose I will close this then. Now there will be an issue people can look back on to know! 😁
